client/web/src/repo/RepoHeader.tsx

```diff
@@ -251,7 +251,28 @@ export const RepoHeader: React.FunctionComponent<Props> = ({ onLifecyclePropsCha
                                 </RepoHeaderActionDropdownToggle>
                                 <MenuList position={Position.bottomEnd}>
                                     {rightActions.map((a, index) => (
-                                        <MenuItem className="p-0" key={a.id || index} onSelect={noop}>
+                                        <MenuItem
+                                            className="p-0"
+                                            key={a.id || index}
+                                            onSelect={noop}
+                                            as={React.forwardRef(
+                                                // Nested ahrefs do not work without preventDefault inside onMouseUp
+                                                // on wrapper div rendered by MenuItem, and is not called by default.
+                                                // https://reach.tech/menu-button/#menuitem-props
+                                                ({ children, onMouseUp, ...rest }, ref) => (
+                                                    <div
+                                                        {...rest}
+                                                        ref={ref}
+                                                        onMouseUp={e => {
+                                                            e.preventDefault()
+                                                            onMouseUp?.(e)
+                                                        }}
+                                                    >
+                                                        {children}
+                                                    </div>
+                                                )
+                                            )}
+                                        >
```

Comment:
```suggestion
                                        <MenuItem
                                            className="p-0"
                                            key={a.id || index}
                                            onSelect={noop}
                                            // Nested ahrefs do not work without preventDefault inside onMouseUp
                                            // on wrapper div rendered by MenuItem, and is not called by default.
                                            // https://reach.tech/menu-button/#menuitem-props
                                            onMouseUp={event => event.preventDefault()}
                                        >
```

WDYT to just use MenuItem.onMouseUp prop?

Reply:
resolved.